packages/forma-36-react-components/src/components/Modal/ModalConfirm/ModalConfirm.tsx

```diff
@@ -56,6 +56,11 @@ export type ModalConfirmProps = {
    */
   isConfirmLoading?: boolean;
 
+  /**
+   * Are modals highter that viewerport allowed
```

Comment:
Couple of typos:

Fixed version: `Are modals higher than viewport allowed`
Alternative suggestion: `When true will allows the Modal to display taller than the viewport`